src/pages/detail-page/subpages/components/NavigatablePanel.tsx

```diff
@@ -133,10 +177,9 @@ const NavigatablePanel: React.FC<NavigatablePanelProps> = ({
       </Grid>
       <Grid
         item
-        container
         md={9}
         ref={scrollableSectionRef}
-        sx={{ height: "500px", overflowY: "auto" }}
+        sx={{ height: panelVisibleHeight + "px", overflowY: "auto" }}
         onScroll={(event) => {
           const scrollPosition = event.currentTarget.scrollTop;
           debouncee(scrollPosition);
```

Comment:
consider renaming this deliberately incorrect spelling, make it clearer, eg. debouncedScrollHandler  

also I noticed that the current block to initialise `debouncee` appears to be initialised on every new renders unnecessarily.  Initialising it inside useEffect to ensure it runs only once.

you should also have a strategy to cancel any debounce timeouts to avoid memory leaks

https://stackoverflow.com/questions/29012922/how-to-cancel-a-debounced-function-after-it-is-called-and-before-it-executes

so changes may look like

```javascript
  const debouncedScrollHandler = useRef<((number: number) => void) | null>(null);

  useEffect(() => {
    debouncedScrollHandler.current = _.debounce((scrollPosition: number) => {
      setPosition(scrollPosition);
    }, 300);

    return () => {
      debouncedScrollHandler.current?.cancel();
    };
  }, []);

  const handleScroll = (event) => {
    if (debouncedScrollHandler.current) {
      debouncedScrollHandler.current(event.currentTarget.scrollTop);
    }
  };

// and then in component's onScroll, you can use onScroll={handleScroll}
```